### PR TITLE
add parcel skeleton code fixes #5

### LINKED ISSUE #5: Add parcel skeleton
Let's add a basic parcel/react skeleton with a hello world component, look at: 

https://github.com/reicheltp/parcel-react-example

for an example but move index.html/index.js into /src

don't forget to update gitignore!

### PR DESCRIPTION
**Pull request by zyjenny**

### CODE REVIEW

File: .babelrc [hunk 1]
```diff
@@ -0,0 +1,5 @@
+{
+  "presets": [
+    "@babel/preset-env", "@babel/preset-react"
+  ]
+}
```
Comment (jaredkerim) [inline .babelrc]: Get outta here newline!

File: .gitignore [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+.vuepress/dist
+
+# Serverless directories
+.serverless/
+
+# FuseBox cache
+.fusebox/
+
+# DynamoDB Local files
+.dynamodb/
```
Comment (jaredkerim) [inline .gitignore]: Shoo newline!

File: package.json [hunk 3]
```diff
@@ -0,0 +1,19 @@
+{
+  "name": "parcel-react",
```
Comment (jaredkerim) [inline package.json]: Change this to "avocado"

File: package.json [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+  "version": "1.0.0",
+  "main": "index.js",
+  "license": "MIT",
```
Comment (jaredkerim) [inline package.json]: This needs to change to MPL (for Mozilla Public License)

File: package.json [hunk 5]
```diff
@@ -0,0 +1,19 @@
...
+  "scripts": {
+    "start": "parcel src/index.html"
+  },
+  "devDependencies": {
+    "@babel/core": "^7.4.4",
+    "@babel/preset-react": "^7.0.0",
+    "babel-preset-react": "^6.24.1",
```
Comment (jaredkerim) [inline package.json]: Do we need both of these babel preset reacts?

File: src/index.html [hunk 6]
```diff
@@ -0,0 +1,6 @@
+<html>
+<body>
+  <div id="root"></div>
+  <script src="./index.js"></script>
+</body>
+</html>
```
Comment (jaredkerim) [inline src/index.html]: Newlines haunt me in my sleep

File: src/index.js [hunk 7]
```diff
@@ -0,0 +1,10 @@
+import React from 'react'
+import * as ReactDOM from 'react-dom'
+
+const App = () => {
+  return (
+    <h1>Hello World!</h1>
```
Comment (jaredkerim) [inline src/index.js]: 👋

File: .babelrc (hunk 1 above)
Comment (zyjenny) [reply .babelrc]: I just realized that little red symbol means "no newline at end of file". Shall I add a newline to get rid of it or keep as is? 🥑

Comment (glasserc) [reply .babelrc]: Just a drive-by review, but I think @jaredkerim means that you should put a newline to get rid of it. I'd also encourage you to see if your text editor has options for automatically ensuring that your files have exactly 1 newline at EOF, making sure no line has trailing whitespace, etc.

**Pull request merged**